src/hooks/useKeyPress.ts

```diff
@@ -69,13 +69,14 @@ export default (keyCode: KeyCode | null = null, options: UseKeyPressOptions = {
       options?.target?.addEventListener('keydown', downHandler as EventListenerOrEventListenerObject);
       options?.target?.addEventListener('keyup', upHandler as EventListenerOrEventListenerObject);
       options?.target?.addEventListener('blur', resetHandler);
+      options?.target?.addEventListener('mouseleave', resetHandler);
```

Comment:
Added a new event `mouseleave` because `blur` is not getting triggered when we click outside of the document like changing the browser tab etc.